x-pack/plugins/cases/public/components/create/flyout/use_cases_add_to_new_case_flyout.tsx

```diff
@@ -23,6 +23,12 @@ export const useCasesAddToNewCaseFlyout = (props: AddToNewCaseFlyoutProps = {})
   const { dispatch } = useCasesContext();
   const casesToasts = useCasesToast();
 
+  const submitFromFlyout = useCallback(() => {
+    dispatch({
+      type: CasesContextStoreActionsList.SUBMIT_CASE_FROM_FLYOUT,
+    });
+  }, [dispatch]);
```

Comment:
> If it is really mandatory to provide that functionality to the users and we cannot find UX alternatives, I think we should change the way you try to submit the case. Instead of passing a prop and using useEffects, cases can provide a callback to call when the user hits the next button.

@cnasikas I did this through a callback as seen here. However, we are outside of the form context. My solution was to update the cases context redux state and inject the state into the flyout props from `getCreateCaseFlyoutLazyNoProvider` in [cases_global_components.tsxL17-L20](https://github.com/elastic/kibana/pull/144249/files#diff-043d5f9897ef2b41f1480f56718d74be02b75e2ee0d9000d42b9a6e4bfc24129R17-R20). How would you suggest to do this instead?